tests/std/tests/VSO_1775715_user_defined_modules/test.cpp

```diff
@@ -3,6 +3,13 @@
 
 // Note: To properly test the fix for VSO-1775715, don't include any headers here.
 
+// TRANSITION, Windows SDK 10.0.22621.0 causes this test to fail for ARM64EC with:
+// "error LNK2019: unresolved external symbol fabsf referenced in function #fabsf$exit_thunk (EC Symbol)"
+// Windows SDK 10.0.26100.0 will avoid that error, but we'll need to investigate why user.ixx emits:
+// "error C2678: binary '==': no operator found which takes a left-hand operand of type 'const std::string'
+// (or there is no acceptable conversion)"

```

Comment:
This error C2678 turns out to be unaffected by architecture or Windows SDK version, and affected by the execution charset: error C2678 manifests if and only if the execution charset is not self-synchronizing (e.g. cp932, cp936). With cp1252 or UTF-8, the test compiles successfully.